CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 - On iOS, send events `WindowEvent::Occluded(false)`, `WindowEvent::Occluded(true)` when application enters/leaves foreground.
 - **Breaking** add `Event::MemoryWarning`; implemented on iOS/Android.
 - On Wayland, support `Occluded` event with xdg-shell v6
+- Implement `AsFd`/`AsRawFd` for `EventLoop<T>` on X11 and Wayland.
```

Comment:
Maybe it would be nice to implement this for `EventLoopWindowTarget` as well?

Reply:
I'm not sure that it worth it, basically you want to get `fd` outside the `run` closure, since inside it it's weird, since your event loop is external?

Like it doesn't make any sense why anyone would do that...

Reply:
Alright, we can put a pin in that for now